### ISSUE TITLE
ERROR: ChefVault::Exceptions::SecretDecryption: is encrypted for you, but your private key failed to decrypt the contents.  (if you regenerated your client key, have an administrator of the vault run 'knife vault refresh')

### ISSUE THREAD

**Issue by naveenbuild** [chef/chef-vault#392]
Hello,

I am trying to setup a new Chef Automate server on AWS OpsWorks. While I was able to create the chef-vault and vault item successfully, getting the below error if I try to run "knife vault show VAULT VAULT_ITEM". 

ERROR: ChefVault::Exceptions::SecretDecryption: VAULT/VAULT_ITEM is encrypted for you, but your private key failed to decrypt the contents.  (if you regenerated your client key, have an administrator of the vault run 'knife vault refresh')

Tried running knife vault refresh VAULT VAULT_ITEM but that is also returning the same error. Any help would be appreciated!

## Software Version
Chef Vault: 4.1.5

**Comment by vkarve-chef** [comment]
Hi @naveenbuild could you please confirm you have admin role on the Chef Infra Server so that the `knife vault refresh` run is effective?

**Comment by naveenbuild** [comment]
Hi Vikram,

Thanks for the response. The issue has been resolved.

Yes, I did have the default admin role and it had two public keys added on
the Server but it wasn’t accepting either of the matching ssh private keys.
So I created a new user, gave admin role, created a new set of ssh keys and
saved the public key on the Chef infra server. After that I was able to
perform knife vault operations successfully.

Thanks,
Naveen
